system/modules/core/elements/ContentAlias.php

```diff
@@ -39,7 +39,7 @@ public function generate()
 			return '';
 		}
 
-		// Overwrite the ID
+		/** @var \ContentElement $objElement */
```

Comment:
Aah, that happens later. Maybe move the comment to the place it actually happens…